Artesian/_GMEPublicOffers/QueryParameters/GMEPOfferQueryParameters.py

```diff
@@ -27,27 +24,41 @@ class GMEPOfferQueryParameters:
             purpose: sets the Purpose to be queried.
             page: sets the Page to be queried.
             pageSize: sets the Page size to be queried.
-            baType: sets the BATypes to be queried."""
+            baType: sets the BATypes to be queried.
+    """
 
 
-    def __init__(self, scope: Scope, extractionRangeSelectionConfig: ExtractionRangeConfig, status: Status, unitType: UnitType, generationType: GenerationType, operator: str, unit: str, zone: Zone , market: Market, purpose: Purpose, page: int, pageSize: int, baType: BaType)  -> None : 
+    def __init__(self, scope: Scope, 
+                       extractionRangeSelectionConfig: ExtractionRangeConfig, 
+                       status: Status, 
+                       unitType: UnitType, 
+                       generationType: GenerationType, 
+                       operator: str, 
+                       unit: str, 
+                       zone: Zone , 
+                       market: Market, 
+                       purpose: Purpose, 
+                       page: int, 
+                       pageSize: int, 
+                       baType: BaType)  -> None : 
```

Comment:
it's strange to see page&pageSize in the middle. I see that was already like that before.
Please reorder them placing page/pageSize as first 2 parameters.

Check usages of this initializer and reorder the parameter there too (should be in GMEOfferQuery only).